crates/sui-graphql-rpc/src/extensions/logger.rs

```diff
@@ -123,8 +116,11 @@ impl Extension for LoggerExtension {
         next: NextExecute<'_>,
     ) -> Response {
         let resp = next.run(ctx, operation_name).await;
+        let session_id = ctx.data_unchecked::<SocketAddr>().to_string();
+        let query_id = ctx.data_unchecked::<Uuid>().to_string();
```

Comment:
Ascribe the type on the variable, rather than using the "rocket fish" operator.  Also, I think you don't need to convert this to a string -- `tracing` will do this for you.
```suggestion
        let session_id: SocketAddr = ctx.data_unchecked();
        let query_id: Uuid = ctx.data_unchecked();
```

Reply:
It will not work without `to_string()`.

Reply:
The piece that was missing here, was that you need to use a `%` sigil to get it to `Display` the type, details here:

https://docs.rs/tracing/latest/tracing/#recording-fields

Reply:
Thanks, that's quite inconspicuous! I will put up another PR to fix this.